Snakemake/workflow/rules/prepare_files_for_tsinfer.smk

```diff
@@ -66,19 +71,28 @@ rule extract_vcf_pos:
 
 rule match_ancestral_vcf:
     input:
+<<<<<<< HEAD
+        vcfPos = rules.extract_vcf_pos.output.file,
+        ancestral = "AncestralAllele/AncestralAllele_Vcf.txt",
+        major = rules.get_major.output
+    output: 
+        file = temp(f'{vcfdir}/AncestralVcfMatch{{chromosome}}.txt'),
+=======
         vcfPos=rules.extract_vcf_pos.output,
         ancestral="AncestralAllele/AncestralAllele_Vcf.txt",
         major=rules.get_major.output
     output: temp('AncestralVcfMatch{chromosome}.txt')
+>>>>>>> 27fdb4deb7416d7d35cf7bb91ea51acbeeb1a66a
     params:
         chrNum = lambda wc: wc.get("chromosome")[3:],
-        ancestral_sites = '{chromosome}.aa'
+        ancestral_sites = f'{vcfdir}/{{chromosome}}.aa',
+        tmp_file = f'{vcfdir}/{{chromosome}}.tmp',
```

Comment:
Why do we need this temp_file?

Reply:
This is to allocate the output of the for loop. At the end it is converted into the output. 
(I've just renamed the tmp files so there's no conflict when running multiple chromosomes at once. I actually don't think we need this tmp file because what is being done with the last awk command after the loop can be done with the sed command cemented out, if I understand correctly what is happening. )

Reply:
Yeah, I think we couldn't have done it without, but it's just an intermediate file. I think it would be better to move to to the "output" instead of "params" and mark it as tmp()